sycl/doc/extensions/proposed/sycl_ext_oneapi_annotated_arg.asciidoc

```diff
@@ -173,163 +169,45 @@ class annotated_arg {
 `annotated_arg` is a class template, parameterized by the type of the underlying
 allocation `T` and a list of associated properties specified by `PropertyListT`.
 
+`T` can be any type except the following types or a structure containing one of
+the following types: sycl::accessor, sycl::stream, sycl::sampler, and
+sycl::half_type.
+
 The properties supported with `annotated_arg` may be defined in
 separate extensions.
 
 The section below describes the constructors and member functions for
 `annotated_arg`.
 
+The template parameter `T` in the definition of `annotated_arg` template below
+must be a device copyable type or a legal parameter type as defined by the SYCL
+specification.
```

Comment:
Kind of a nit, but it is sufficient to say:

> The template parameter `T` in the definition of `annotated_arg` template below
must be a legal parameter type as defined by the SYCL specification.

The SYCL specification already says that "device copyable" types are legal kernel arguments.

Reply:
Updated!